tests/basic_functionality.rs

```diff
@@ -257,3 +258,91 @@ fn report_with_j2_max_and_j1_max_minus_1_generates_2_tcns() {
     let recomputed_tcns = report.temporary_contact_numbers().collect::<Vec<_>>();
     assert_eq!(recomputed_tcns.len(), 2);
 }
+
+fn tcn_generation_test_helper(
```

Comment:
It should be clear what the helper does from the name (if not possible at least a comment).